assets/src/apps/authoring/components/EditingCanvas/AuthoringActivityRenderer.tsx

```diff
@@ -37,11 +37,6 @@ const AuthoringActivityRenderer: React.FC<AuthoringActivityRendererProps> = ({
 
   const selectedPartId = useSelector(selectCurrentSelection);
 
-  if (!activityModel.authoring || !activityModel.activityType) {
```

Comment:
This was hitting the conditionally run hooks rule, hooks must be at the top of the component definition and run every render. (https://reactjs.org/docs/hooks-rules.html)